.github/workflows/build.yml

```diff
@@ -26,31 +26,103 @@ jobs:
           containerfiles: |
             ./Containerfile
           labels: |
             ${{ env.EXPIRATION_LABEL }}
             quipucords.backend.git_sha=${{ github.sha }}
 
-      - name: Push To quay.io
+      - name: Push Amd64 To quay.io
         # Forks that do not set secrets and override the variables may fail this step.
         uses: redhat-actions/push-to-registry@v2
         with:
           image: ${{ env.IMAGE_NAME }}
-          tags: ${{ steps.build-image.outputs.tags }}
+          tags: ${{ steps.build-image-amd64.outputs.tags }}
           registry: ${{ env.REGISTRY }}
           username: ${{ secrets.QUAYIO_USERNAME }}
           password: ${{ secrets.QUAYIO_PASSWORD }}
-        continue-on-error: true
+
+  build-arm64:
+    runs-on: ubuntu-24.04-arm
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0  # fetches all commits/tags
+
+      - name: Define Release Tag Arm64
+        id: define-release-tag-arm64
+        shell: bash
+        run: echo "RELEASE_TAG_ARM64=$([[ ${GITHUB_REF_NAME} =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]] && echo ${GITHUB_REF_NAME%.[0-9]*}-arm64 || echo)" >> "$GITHUB_ENV"
+
+      - name: Build Arm64 quipucords image
+        id: build-image-arm64
+        uses: redhat-actions/buildah-build@v2
+        with:
+          image: ${{ env.IMAGE_NAME }}
+          archs: arm64
+          tags: ${{ env.STABLE_TAG }}-arm64 ${{ env.STABLE_TAG == 'main' && 'latest-arm64' || '' }} ${{ contains( env.RELEASE_TAG_ARM64 , '.' ) && env.RELEASE_TAG_ARM64 || '' }}
+          containerfiles: |
+            ./Containerfile
+          labels: |
+            ${{ env.EXPIRATION_LABEL }}
+            quipucords.backend.git_sha=${{ github.sha }}
+
+      - name: Push Arm64 To quay.io
+        # Forks that do not set secrets and override the variables may fail this step.
+        uses: redhat-actions/push-to-registry@v2
+        with:
+          image: ${{ env.IMAGE_NAME }}
+          tags: ${{ steps.build-image-arm64.outputs.tags }}
+          registry: ${{ env.REGISTRY }}
+          username: ${{ secrets.QUAYIO_USERNAME }}
+          password: ${{ secrets.QUAYIO_PASSWORD }}
+
+  build:
+    runs-on: ubuntu-latest
+    needs: [build-amd64, build-arm64]
+
+    steps:
+      - name: Define Release Tag
+        id: define-release-tag
+        shell: bash
+        run: echo "RELEASE_TAG=$([[ ${GITHUB_REF_NAME} =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]] && echo ${GITHUB_REF_NAME%.[0-9]*} || echo)" >> "$GITHUB_ENV"
+
+      - name: Define Image Tag
+        id: define-image-tag
+        shell: bash
+        run: echo "IMAGE_TAG=${{ env.STABLE_TAG == 'main' && 'latest' || contains( env.RELEASE_TAG , '.' ) && env.RELEASE_TAG || env.STABLE_TAG }}" >> "${GITHUB_ENV}"
+
+      - name: Push Manifest
+        id: push-manifest
+        uses: pixelfederation/gh-action-manifest-tool@v0.1.7
+        with:
```

Comment:
Don't we need a conditional here? otherwise when `push-versioned-manifest` is invoked, this will run as well 🤔

Reply:
🤔 I guess if we only want the x.y.z tag we can do an `if: !contains( env.RELEASE_TAG, '.' )` here

Reply:
had to remove that back, this puppy handled x.y 
In latest changes, I've added comments as to which tags are handles in which task.